MonkeyBot/Common/XYChart.cs

```diff
@@ -45,6 +45,8 @@ public XYChart(int width, int height, int borderMargin)
             chartWidth = width;
             chartHeight = height;
             axisBorder = borderMargin;
+            AxisX = axisX;
```

Comment:
does this not complain about AxisX/Y being possibly null?

Reply:
Nope, because you cannot pass null into the constructor without getting a similar warning. :-)